src/outlineView.ts

```diff
@@ -135,6 +135,19 @@ const supportedTypes = [
   "variable",
 ];
 
+const symbolMap = {
+  class: '\\e600',
```

Comment:
I tried without escaping too
```suggestion
  class: '\e600',
```